converter/COLLADA2GLTF/main.cpp

```diff
@@ -82,6 +82,7 @@ static const OptionDescriptor options[] = {
 	{ "h",              no_argument,        "-h -> help" },
 	{ "r",              no_argument,        "-r -> verbose logging" },
 	{ "e",				no_argument,		"-e -> embed resources (bin, shaders, available textures) in glTF file" },
+    { "P",              no_argument,        "-P -> pretty print glTF JSON" },
```

Comment:
I'm not sure if we should change the default print style since this is the open source converter. @pjcozzi what do you think?

I'm thinking we should have it default to pretty print. Give an option to ugly-print it (I wouldn't call it ugly-print, maybe minimize size or something like that).